src/Adapters/EventSource.Netstandard13/EventSourceListeningRequest.cs

```diff
@@ -56,7 +51,7 @@ public override bool Equals(object obj)
                 return false;
             }
 
-            return this.Name == other.Name && this.Level == other.Level && this.Keywords == other.Keywords;
+            return this.Name == other.Name && this.Level == other.Level && this.Keywords == other.Keywords && this.PrefixMatch == other.PrefixMatch;
```

Comment:
should you define `Equals` for base class? If so - call it's `Equals` first.

If you do not need `Equals` for base class, this `Equals` is also unnecessary

Reply:
Good point. Will fix in next iteration.